puppet/modules/slave/templates/copr.erb

```diff
@@ -0,0 +1,7 @@
+[copr-cli]
+login = <%= @copr_login %>
```

Comment:
Where do these variables come from? Global variables?

Reply:
I *think* so, but hoping @GregSutcliffe can confirm

Reply:
Foreman can hold secure variables and pas them in as smart class parameters, yes. Just set sane defaults here.